packages/documentation/copy/en/handbook-v2/Type Manipulation/Generics.md

```diff
@@ -248,7 +248,7 @@ console.log(stringNumeric.add(stringNumeric.zeroValue, "test"));
 
 Just as with interface, putting the type parameter on the class itself lets us make sure all of the properties of the class are working with the same type.
 
-As we covered in [our section on classes](/docs/handbook/2/classes.html), a class has two sides to its type: the static side and the instance side.
+As cover in [our section on classes](/docs/handbook/2/classes.html), a class has two sides to its type: the static side and the instance side.
```

Comment:
```suggestion
As we will cover in [our section on classes](/docs/handbook/2/classes.html), a class has two sides to its type: the static side and the instance side.
```

There isn't really a tense neutral way to say this in English ("see also" maybe) but I think switching the tense is reasonable

Reply:
I missed the "we" between the "As" and "cover". See if that has a different reading impact. Just pushed a change to the branch for that.

Reply:
I agree, I like your version 👍🏻